.github/workflows/long-running-azure.yaml

```diff
@@ -45,40 +45,55 @@ on:
     branches:
       - main
     paths:
-      - '.github/workflows/long-running-azure.yaml'
+      - ".github/workflows/long-running-azure.yaml"
 
 env:
   # Go version
-  GOVER: '^1.21'
+  GOVER: "^1.21"
```

Comment:
Please use single-quote. if you want to change it to double-quote, please create the separate pr to change all workflow file.

Reply:
The file uses both single and double quotes. I reverted to the original version. I will refactor the files in a new PR.